src/masternode-budget.cpp

```diff
@@ -81,26 +82,34 @@ bool IsBudgetCollateralValid(const uint256& nTxCollateralHash, const uint256& nE
         - nTime is never validated via the hashing mechanism and comes from a full-validated source (the blockchain)
     */
 
-    int conf = GetIXConfirmations(nTxCollateralHash);
+    nConf = GetIXConfirmations(nTxCollateralHash);
```

Comment:
We can remove this `GetIXConfirmations` from here. It will always return 0.

Reply:
Yeah because IX is disabled for now.
But didn't want to mix IX remnants removal in this refactoring PR. Better to address that in a dedicated PR.